src/engine.h

```diff
@@ -1,4 +1,20 @@
-// Define this class here.
+// Declare this class here
+#include<iostream>
 class RandomEngine {
+    private:
+         uint8_t length=0;
```

Comment:
This is not the way to initialize variables, all class members must be initialized via constructors